File: Python/1012.py
a, b, c = input().split()
a, b, c = float(a), float(b), float(c)

pi = 3.14159

triangle = (a * c) / 2
circle = pi * pow(c, 2)
trapezium = (1/2) * (a + b) * c
square = pow(b, 2)
rectangle = a * b

print('''TRIANGULO: %.3f
CIRCULO: %.3f
TRAPEZIO: %.3f
QUADRADO: %.3f
RETANGULO: %.3f''' % (triangle, circle, trapezium, square, rectangle))

File: C/1012.c
#include <stdio.h>

int main()
{

    double a, b, c;
    double areaTri, areaC, areaTra, areaQ, areaRet;
    double pi = 3.14159;
    scanf("%lf %lf %lf", &a, &b, &c);

    areaTri = a * c / 2;
    areaC = pi * c * c;
    areaTra = (a + b) * c / 2;
    areaQ = b * b;
    areaRet = a * b;

    printf("TRIANGULO: %.3f\n", areaTri);
    printf("CIRCULO: %.3f\n", areaC);
    printf("TRAPEZIO: %.3f\n", areaTra);
    printf("QUADRADO: %.3f\n", areaQ);
    printf("RETANGULO: %.3f\n", areaRet);

    return 0;
}